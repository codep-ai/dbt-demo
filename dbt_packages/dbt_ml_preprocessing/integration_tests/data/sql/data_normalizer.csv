id_col,col1,col2,col3,col4
1,4,1,2,2
2,1,3,9,3
3,5,7,5,1
4,1,2,3,4
5,1,10,15,20
4,6,6,6,6
7,1,2,3,4
8,9,8,7,6
9,5,5,10,10
10,5,5,-5,-5