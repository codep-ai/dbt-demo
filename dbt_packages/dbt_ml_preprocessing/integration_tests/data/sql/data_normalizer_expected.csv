id_col,col1,col2,col3,col4,col1_normalized,col2_normalized,col3_normalized,col4_normalized
1,4,1,2,2,0.8,0.2,0.4,0.4
2,1,3,9,3,0.1,0.3,0.9,0.3
3,5,7,5,1,0.5,0.7,0.5,0.1
4,1,2,3,4,0.18257418583505536,0.3651483716701107,0.5477225575051661,0.7302967433402214
5,1,10,15,20,0.037113480951260276,0.37113480951260275,0.5567022142689041,0.7422696190252055
4,6,6,6,6,0.5,0.5,0.5,0.5
7,1,2,3,4,0.18257418583505536,0.3651483716701107,0.5477225575051661,0.7302967433402214
8,9,8,7,6,0.5934424260562083,0.5275043787166296,0.4615663313770509,0.3956282840374722
9,5,5,10,10,0.31622776601683794,0.31622776601683794,0.6324555320336759,0.6324555320336759
10,5,5,-5,-5,0.5,0.5,-0.5,-0.5
